gateway/handler/signing/signingHandler.ts

```diff
@@ -12,7 +13,7 @@ import { signAndEncodeResponse } from "./signAndEncodeResponse";
  * @param configEntry - The signing configuration entry.
  * @returns The signed and encoded response.
  */
-export async function signingHandler(calldata: string, resolverAddr: string, configEntry: SigningConfigEntry) {
+export async function signingHandler(calldata: string, resolverAddr: string, configEntry: SigningConfigEntry, logger: Logger) {
```

Comment:
Can't we make logger a global object instead of passing it into every function call ? 

global.logger.debug()